config/config.go

```diff
@@ -1369,9 +1371,20 @@ func getGoGCEnv() int {
 	return DefaultRuntimeConfig.GoGC
 }
 
+type translationStrategyOption string
+
+var (
+	// NoUTF8EscapingWithSuffixes will keep UTF-8 characters as they are, units and type suffixes will still be added.
```

Comment:
I thought the two options were going to be "no alteration at all" and "underscoreescapingwithsuffixes"?

Reply:
(is this because type/unit is differentiating?)